kernel/signal.c

```diff
@@ -435,6 +427,16 @@ static long _handle_one_signal(
     }
     else if (action->handler == (uint64_t)SIG_IGN)
     {
+        if (__myst_kernel_args.strace_config.trace_syscalls ||
+            __myst_kernel_args.trace_errors)
+        {
+            myst_eprintf(
```

Comment:
I wonder whether we should start using the color encoded syslog feature for these kinds of messages (since they can be suppressed by setting the logging level).

Reply:
lets deal with coloring of different messages and filtering in another PR